service/core/serverObj/v2ray.go

```diff
@@ -294,9 +294,15 @@ func (v *V2Ray) Configuration(info PriorInfo) (c Configuration, err error) {
 				core.StreamSettings.TCPSettings = &tcpSetting
 			}
 		case "h2", "http":
-			core.StreamSettings.HTTPSettings = &coreObj.HttpSettings{
-				Path: v.Path,
-				Host: strings.Split(v.Host, ","),
```

Comment:
However, I think if the Host is "", this item will be omitted automatically. So no need to change here. What do you think?

Reply:
Thanks for the comments! I will test it tomorrow. 
At the time, I was not sure if [ "" ] is treated as empty or not, to me it was not obvious.
However, I have a feeling that it might be a good idea that we should avoid such tricks. But it is your call.